app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -56,7 +56,7 @@ KeyPair generateKeyPair(final IKey.Types keyType) {
      */
     boolean validateName(final EditText input) {
         final String name = input.getText().toString();
-        if (this.isValidName(name)) {
```

Comment:
Because isValidName doesn't belong to this, but to CreateProfileHelper. The method is static